federation/pkg/kubefed/init/init.go

```diff
@@ -501,9 +501,9 @@ func createControllerManager(clientset *client.Clientset, namespace, name, cmNam
 							Command: []string{
 								"/hyperkube",
 								"federation-controller-manager",
-								"--master=https://federation-apiserver",
+								fmt.Sprintf("--master=https://%s", svcName),
 								"--kubeconfig=/etc/federation/controller-manager/kubeconfig",
-								"--dns-provider=gce",
+								"--dns-provider=google-clouddns",
```

Comment:
@quinton-hoole @irfanurrehman fixed this in PR #36492. So I am removing this change from here.